dts/arm/st/stm32f7.dtsi

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2018 Yurii Hamann
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include <st/mem.h>
+#include <dt-bindings/clock/stm32_clock.h>
+#include <dt-bindings/gpio/gpio.h>
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m7";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 DT_SRAM_SIZE>;
+	};
+
+	flash0: flash@8000000 {
+	compatible = "soc-nv-flash";
+	label = "FLASH_STM32";
+	reg = <0x08000000 DT_FLASH_SIZE>;
+	write-block-size = <1>;
+	};
+
+	soc {
+		rcc: rcc@40023800 {
+			compatible = "st,stm32-rcc";
+			clocks-controller;
+			#clocks-cells = <2>;
```

Comment:
Rename to "#clock-cells" (This has changed recently, there is now a warning associated).